### PR TITLE
Copied version of a project made during the admission trials

### PR DESCRIPTION
**Pull request by zkr024**
project made using HTML and CSS.

### CODE REVIEW

**Comment by zkr024** [comment]
Please make sure to update the README.md file. It's necessary for it to have more information about your project. Feel free to use this [tempale](https://github.com/zkr024/portfolio/tree/headline-section#readme). Make sure to edit it with your information.

**Comment by zkr024** [comment]
Please remember to set up Linters in your project. This will allow you to identify and correct those areas in your code that need some changes to accomplish code standard procedures. For more information on how to set it up, please visit this [link](https://github.com/microverseinc/linters-config)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @zkr024 👋, 

Good job so far on this special project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ]  You did a great job on this special project performing code-review, But there is a little issue with the comments you used single comments, According to the requirements you should use inline comments, Please use inline comments for your code review. When commenting on code instead of "add single comment" select the "start a review" option. Please read this article about [How to perform code reviews](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zkr024** [comment]
@thecodechaser Hi there. I'm not sure what I did but all the comments that I left are gone. Should I just select the range of the lines that need changes and add the comments once again? Thanks for your help

File: index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            <div><input type="text" placeholder="Search" class="searchbar"></div>
+            <input type="button" value="" class="buttons"> 
+        </form>
+
+        <form class="menu">
+                <input type="button" value="" class="buttons option">
+                <input type="button" value="" class="buttons option">
+                <input type="button" value="" class="buttons option">           
+                <input type="button" value="" class="buttons option account"> 
+        </form>
```
Comment (zkr024) [inline index.html:39]: Please use the "nav" tag to cover this area for code semantic purposes.

**Comment by thecodechaser** [comment]
@zkr024 Hello, Yes please select lines of code where you want to request changes and start the review.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    </header>
+
+    <div class="page">
+
+        <article>
+            <div class="media">
+                <video src="video/RentaGirlfriend _Opening.mp4" controls></video>
+                <p class="title">Rent a Girlfriend opening</p>
```
Comment (zkr024) [inline index.html:47]: Please remove the space from the video name in the src area. That is necessary to pass the Linters test.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                        <img class="img_owner" src="Images/sasuke.png" alt="sasuke">
+                    </div>
+                    <ul class="credit_info">
+                        <li class="owner">Sasuke</li>
+                        <li class="owner subscriptions">5k subscriptors</li>
+                        <li class="owner data">Welcome to my anime world channel</li>
+                    </ul>
+                </div>
+            </div>
+        </article>
```
Comment (zkr024) [inline index.html:68]: Is necessary to use an "h" tag in the article to inform which is the section of your page.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+            <ul class="minivideos">
+                <li class="vidinfo"><img class="minimages" src="Images/images (1).jpeg" alt="img taken from google">
+                    <div class="mindetails">
+                        <a class="mintitle" href="#">Dragon Ball</a>
+                        <ul class="mininfo">
+                            <li>Uploaded by</li>
+                            <li>Visualization</li>
+                            <li>Date</li>
+                        </ul>
```
Comment (zkr024) [inline index.html:87]: Inside the list, you took some images. Please remove the spaces from the names of the images to accomplish the Linters standard procedures. Please apply these changes to the rest of the images.

**Comment by zkr024** [comment]
@thecodechaser Thank you. I will do that!

File: index.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                    </div>
+                </li>
+            </ul>
+
+        </aside>
+
+    </div>
+    
+</body>
+</html>
```
Comment (zkr024) [inline index.html:177]: Please add an empty line at the end of your code. This is again for code semantic purposes.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+}
+.links{
+    list-style: none;
+    display: flex;
+    overflow-x: auto;
+    height: 60px;
+    align-items: center;
+    margin-bottom: 15px;
+       
+}
```
Comment (zkr024) [inline style.css:102]: Inside this selector, you left an empty line. Please make sure to remove it. Once you set up Linters, you will be able to find and correct these situations.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (zkr024) [inline style.css:217]: There are several empty lines at the end of your code, please make sure to leave only one.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @zkr024 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**